src/proto/mgmt/pool.proto

```diff
@@ -206,6 +207,8 @@ message PoolQueryResp {
 	uint32 total_nodes = 9; // total nodes in pool
 	uint32 version = 10; // latest pool map version
 	uint32 leader = 11; // current raft leader
+	int32 info_bit = 12; // Pool query bit set
```

Comment:
suggestion: `bool ranks_enabled`

Reply:
Or you could go with the idea I mentioned above, having something like:
```suggestion
	repeated uint32 enabled_ranks = 12;
	repeated uint32 disabled_ranks = 13;
```

Whether that's a good approach depends on whether we expect pools to be able to have a mix of enabled and disabled ranks.

Reply:
To be honest, my first intention was to do something such as this to decouple the control-plane and the engine. As explained before, I finally choose to keep the bitfield in the RPC protocol to easy handle evolution of this bitfield usage: for example to not have to update the protocol if a new bitmask has to be managed or if we have to handle exisiting C bit mask. FMHO, the two solution have their pro and cons. However, if you (i.e. @mjmac and @kjacque) think that decoupled thing is more important, then I will update the RPC protocol as suggested by @kjacque .

Reply:
Yeah, in general we have tried to avoid the pattern where a response includes information about the request because it encourages coupling across layers which is usually not a good design. I think that it would be best to just have two separate fields, one for enabled ranks and one for disabled ranks. It's OK if only one or the other is used right now.

Reply:
I think the decoupling is necessary. A caller of the Go control API should not have to worry about values from the C calls  being used several code layers below. I would prefer to remove those DPI_* values from the Go code altogether, and have the C side drpc handler do the translation.

Reply:
OK, lets go for decoupling. Thanks for feedback :)
- [ ] Decouple RPC protocol from engine stuff